config/vufind/config.ini

```diff
@@ -2156,13 +2156,21 @@ includeSchemaOrgMetadata = true
 
 ; The following two sections control the Alphabetic Browse module.
 [AlphaBrowse]
+
 ; This setting controls how many headings are displayed on each page of results:
 page_size = 20
+
 ; How many headings to show before the match (or the spot where the match
 ; would have been found). Default is 0 for backwards compatibility.
 rows_before = 0
+
+; For topic browse, controls the separator to place between terms when
+; displaying topic headings.
+topic_browse_separator = " -- "
```

Comment:
Since the value configured here differs from the default in the code, should this be commented out as an example? Should the existing default be mentioned in the comments?

Reply:
Gah, whoops.  This was me testing locally and it looked right enough when reviewing the diff that I didn't notice.  I've changed the line to show the default, commented it out and also clarified the leading comment to mention the default.